src/proj4leaflet.js

```diff
@@ -112,6 +112,8 @@
 
 			this.infinite = !this.options.bounds;
 
+			this.minZoom = 0;
+			this.maxZoom = this._scales[i] - 1;

```

Comment:
```suggestion
			this.maxZoom = this._scales[this._scales.length - 1];
```

Reply:
`i` is `NaN` at this point, so I think setting the highest scale makes more sense, unless I'm missing something.